src/legacy/core_plugins/data/public/expressions/expressions_service.ts

```diff
@@ -40,7 +41,9 @@ export class ExpressionsService {
     // eslint-disable-next-line
     const { getInterpreter } = require('../../../interpreter/public/interpreter');
     getInterpreter()
-      .then(setInterpreter)
+      .then(({ interpreter }: { interpreter: IInterpreter }) => {
```

Comment:
is this needed ? (should it be done in master as well ?)

Reply:
Yes, I needed to unwrap in at least one place otherwise our use of ExpressionRenderer would throw an error `cannot find interpreter.interpretAst`